flags.go

```diff
@@ -95,6 +96,11 @@ func defineFlagVar() *FlagVar {
 		&flagVar.logLevel, "log-level", defaultLogLevel,
 		"indicates the current log-level, enter negative values to increase verbosity (e.g. 9)",
 	)
+	flag.BoolVar(&flagVar.enablePurgeFinalizer, "enable-purge-finalizer", true,
```

Comment:
I think default should be false and it should be explicitly enabled in the control-plane config